gradle.properties

```diff
@@ -39,4 +39,7 @@ POM_DEVELOPER_NAME=PayPal Android
 #TODO: needs to be defined
 POM_DEVELOPER_EMAIL=not_defined
 
+cardinalUsername=paypal_sgerritz
+cardinalPassword=AKCp8jQ8tAahqpT5JjZ4FRP2mW7GMoFZ674kGqHmupTesKeAY2G8NcmPKLuTxTGkKjDLRzDUQ
```

Comment:
The Android NXO SDK includes Cardinal 👀  ?

Reply:
@scannillo yes, unfortunatley. Technically, third party integrations dont need it, but the way the repo is designed, it is included. We added a ticket to start to talk to the NXO team to remove that dependency on third party